cmake/modules/hunter_experimental_add_host_project.cmake

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) 2016 Fabian Winnen
+# All rights reserved.
+#
+# Warning:
+#   This module is in experimental stage.
+#   TODO: more disclamers
+#   TODO: notice that we clear some env variables?
+#
+# Adds a project with CMakeList.txt as seperate build with default (host) toolchain
+#
+# Usage:
+#   hunter_experimental_add_host_project(host)
+#
+# sets HUNTER_HOST_ROOT variable to point at host project's binary directory
+message(WARNING "You are using a hunter module in experimental stage:\nhunter_experimental_add_host_project\n")
+
+include(CMakeParseArguments) # cmake_parse_arguments
+
+include(hunter_internal_error)
+include(hunter_status_print)
+include(hunter_status_debug)
+
+function(hunter_experimental_add_host_project)
+  cmake_parse_arguments(_arg "" "" "" ${ARGV})
+
+  # Get subdirectory home
+  list(LENGTH _arg_UNPARSED_ARGUMENTS _len)
```

Comment:
We are in function now so can use pretty names for variables. Let it be `arg` and `len`.